jobs/e2e_node/benchmark-config.yaml

```diff
@@ -1,74 +1,74 @@
 ---
 images:
   cosstable2-resource1:
-    image: cos-stable-73-11647-510-0
+    image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-1
     metadata: "user-data<test/e2e_node/jenkins/gci-init.yaml,gci-update-strategy=update_disabled"
     tests:
       - 'resource tracking for 0 pods per node \[Benchmark\]'
   cosstable2-resource2:
-    image: cos-stable-73-11647-510-0
+    image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-1
     metadata: "user-data<test/e2e_node/jenkins/gci-init.yaml,gci-update-strategy=update_disabled"
     tests:
       - 'resource tracking for 35 pods per node \[Benchmark\]'
   cosstable2-resource3:
-    image: cos-stable-73-11647-510-0
+    image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-1
     metadata: "user-data<test/e2e_node/jenkins/gci-init.yaml,gci-update-strategy=update_disabled"
     tests:
       - 'resource tracking for 105 pods per node \[Benchmark\]'
 
   cosstable2-density1:
-    image: cos-stable-73-11647-510-0
+    image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-1
     tests:
       - 'create 35 pods with 0s? interval \[Benchmark\]'
   cosstable2-density2:
-    image: cos-stable-73-11647-510-0
+    image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-1
     tests:
       - 'create 105 pods with 0s? interval \[Benchmark\]'
   cosstable2-density2-qps60:
-    image: cos-stable-73-11647-510-0
+    image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-1
     tests:
       - 'create 105 pods with 0s? interval \(QPS 60\) \[Benchmark\]'
   cosstable2-density3:
-    image: cos-stable-73-11647-510-0
+    image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-2
     tests:
       - 'create 105 pods with 0s? interval \[Benchmark\]'
   cosstable2-density4:
-    image: cos-stable-73-11647-510-0
+    image: cos-77-12371-251-0
     project: cos-cloud
     machine: n1-standard-1
     tests:
       - 'create 105 pods with 100ms interval \[Benchmark\]'
 
   cosstable1-resource1:
-    image: cos-stable-77-12371-227-0 
+    image: cos-stable-81-12871-103-0
```

Comment:
Can we use cos-81-12871-103-0? 81 is the latest LTS.

gcloud compute images list |grep cos-cloud
cos-69-10895-385-0                                    cos-cloud            cos-69-lts                                    READY
cos-73-11647-534-0                                    cos-cloud            cos-73-lts                                    READY
cos-77-12371-273-0                                    cos-cloud            cos-77-lts                                    READY
cos-81-12871-103-0                                    cos-cloud            cos-81-lts                                    READY
cos-beta-81-12871-44-0                                cos-cloud            cos-beta                                      READY
cos-dev-84-13078-0-0                                  cos-cloud            cos-dev                                       READY
cos-stable-81-12871-103-0                             cos-cloud            cos-stable                                    READY

Reply:
hi roy, 
would it be better to use `image_family: cos-stable`, would that consistently be the most up to date stable release?
and I'm not sure why we do 2 cos releases, but should the 77 one switch to `image_family: cos-77-lts`? I see that used in other places.

Reply:
@bsdnet cos-81 is less expressive than cos-stable-81 in my opinion. there is at least cos-beta-81 from the same milestone.

Reply:
@MHBauer Using image_family cos-77-lts, cos-stable, cos-beta or cos-dev is what I'm planning to do as a next step. This PR is a bugfix. Switching to image_family is an enhancement. I'd prefer not to mix them in one PR.

Reply:
yea, not suggesting a change here, trying to get an understanding from an expert while they may notice.

Reply:
@MHBauer  Yes, we should use cos-77-lts, cos-81-lts, and if the image_family prove working, that will reduce the frequency to update it.

@bart0sh cos-stable-81 is more expressive, but it will be short lived. For Continuous Integration, we can use it. For pre/post submit. We should use latest LTS.  See
https://cloud.google.com/container-optimized-os/docs/resources/support-policy